lib/models/addon-test-app.js

```diff
@@ -32,6 +32,11 @@ AddonTestApp.prototype.create = function(appName, options) {
     .then(() => this);
 };
 
+AddonTestApp.prototype.link = function(appName) {
```

Comment:
I would prefer to call it `linkAddon` or something like that, to disambiguate it a bit more